charts/gcp-managed-cert/templates/cert.yaml

```diff
@@ -0,0 +1,7 @@
+---
+apiVersion: networking.gke.io/v1beta1
+kind: ManagedCertificate
+metadata:
+  name: {{ .Release.Name }}
+  domains:
```

Comment:
Missing `spec:` between `name` and `domains`, I think

Reply:
it's cut pasted from the argo-server chart

Reply:
That one has a `spec:`: https://github.com/broadinstitute/monster-helm/blob/master/charts/argo-server/templates/service.yaml#L19-L25